js/packages/redux/reducers/messenger.reducer.ts

```diff
@@ -121,14 +124,48 @@ export const selectPeerFromPeerID = (state: LocalRootState, peerID: string) => {
 	return peer
 }
 
-// react to events:
-// when i receive PeerConnected event => fill PeerNetworkStatus object with the peerID, the transport et the connected connection status
-// when i receive PeerDisconnected event => delete the PeerNetworkStatus object correspondant to the peerdID
-// when i receive PeerGroupAssociated => fill GroupsDevicesToPeerBucket object with the GroupPK for the id, and for devices fill DeviceToPeerBucket object with the devicePK for the id and the peerID witht the peerID
+// thunk to get a peer from a member
+export const getPeerFromMemberPK = createAsyncThunk(
+	'messenger/gotPeerFromMemberPK',
```

Comment:
```suggestion
	'messenger/getPeerFromMemberPK',
```
I think this is the name of the event fired at the start of the thunk so it's better